<launch>
   
	<arg name="pattern_name" default="speak_for_demo"/>
	<arg name="instance_id" default="default"/>
    <!-- To start on launch of pattern use the flag start:=true -->
	<arg name="start" default="true"/>

    <rosparam file="$(find harmoni_pattern)/config/configuration.yaml" subst_value="True"/>
    <param name="instance_id" value="$(arg instance_id)"/>
    <param name="pattern_name" value="$(arg pattern_name)"/>
    <param name="start" value="$(arg start)"/>
    <node pkg="harmoni_pattern" type="sequential_pattern.py" name="harmoni_pattern" output="screen"  />

</launch>
